loadbalancer/etcd/client_test.go

```diff
@@ -0,0 +1,83 @@
+package etcd_test
+
+import (
+	etcd "github.com/coreos/etcd/client"
+	kitetcd "github.com/go-kit/kit/loadbalancer/etcd"
+	"golang.org/x/net/context"
+)
+
+import "testing"
+
+func TestNoCertificateClient(t *testing.T) {
+	context := context.Background()
```

Comment:
This shadows the context package. Either use `ctx`, or just pass `context.Background()` directly to the various constructors.